rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/UnityFeedbackDescriptionProvider.kt

```diff
@@ -1,13 +1,19 @@
+@file:Suppress("UnstableApiUsage")
+
 package com.jetbrains.rider.plugins.unity
 
 import com.intellij.ide.FeedbackDescriptionProvider
 import com.intellij.openapi.project.Project
+import com.intellij.workspaceModel.ide.WorkspaceModel
 import com.jetbrains.rider.plugins.unity.util.UnityInstallationFinder
+import com.jetbrains.rider.plugins.unity.workspace.getPackages
 
 class UnityFeedbackDescriptionProvider: FeedbackDescriptionProvider {
     override fun getDescription(project: Project?): String? {
         if (project == null) return null
         val version = UnityInstallationFinder.getInstance(project).getApplicationVersion() ?: return null
-        return "Unity: $version"
+        val allPackages = WorkspaceModel.getInstance(project).getPackages()
+        val packageVersion = allPackages.firstOrNull { it.id == "com.unity.ide.rider"}?.version
```

Comment:
You can use `tryGetPackage("com.unity.ide.rider")`. And if it returns `null`, use a default value of `<missing>` or something.

Reply:
Thanks! Resolved.